src/main/resources/resources/logisim/strings/std/std_pl.properties

```diff
@@ -773,7 +773,7 @@ TTL7413 = 7413: podwójna 4-wejściowa bramka NAND (schmitt-trigger)
 TTL7414 = 7414: falownik sześciokątny (schmitt-trigger)
 # ==> TTL74161 =
 # ==> TTL74163 =
-TTL74165 = 74165: 8-bitowy rejestr przesunięcia równoległego do szeregowego
+TTL74165 = 74165: 8-bitowy rejestr przesunięcia równoległego do szeregowego z ładowanie asynchroniczne
```

Comment:
This is a broken translation. Should at least be

    74165: 8-bitowy rejestr przesunięcia równoległego do szeregowego z ładowaniem asynchronicznym

I recommend you roll back all the translations for the languages you do not speak. 

 = 74165: 8-bit parallel-to-serial shift register with asynchronous load

Reply:
fair enough- I was just thinking if I got them close enough by borrowing from the existing translations, a native speaker could correct it without even necessarily knowing the original language

Reply:
question: should I be removing the already existing translations for TTL74165, since they need to be re-done?

Reply:
or is putting "(asynchronous load)" in English after each one a good solution?

Reply:
No, no English add-ons for non English string please. There's always original English string which can be used as reference if needed.